moon-store-mac-os/moon-store-mac-os/Helpers/Components/MSImagePicker/MSImagePickerView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  MSImagePickerView.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 1/2/25.
+//
+
+import SwiftUI
+import _PhotosUI_SwiftUI
+
+private enum Constants {
+    static let imageSize: CGFloat = 200
+    static let cornerRadius: CGFloat = 6
+    static let strokeLineWidth: CGFloat = 2
+}
+
+struct MSImagePickerView: View {
+    @StateObject private var viewModel: MSImagePickerViewModel = .init()
+    
+    @Binding private var selectedImage: Image?
```

Comment:
Remove unnecessary space.
```suggestion
    @StateObject private var viewModel: MSImagePickerViewModel = .init()
    @Binding private var selectedImage: Image?
```